frameworks/digital-outcomes-and-specialists/questions/briefs/questionAndAnswerSessionDetailsSpecialists.yml

```diff
@@ -0,0 +1,46 @@
+id: questionAndAnswerSessionDetails
+name: Question and answer session details
+question: Question and answer session details
+question_advice: |
+  <br>
```

Comment:
Have had a chat with Ralph, it's there to make the optional flag on the rendered page look less like a heading for the text. He's agreed to remove it and he'll add it to the design technical debt.